pkg/agent/openflow/pipeline.go

```diff
@@ -898,6 +947,43 @@ func (c *client) arpNormalFlow(category cookie.Category) binding.Flow {
 		Done()
 }
 
+// ipv6Flows generates the flows to allow IPv6 packets from link-local addresses and
+// handle multicast packets, ND solicitation and ND advertisement packets properly.
+func (c *client) ipv6Flows(category cookie.Category) []binding.Flow {
```

Comment:
Name it gatewayIPv6Flows?

Reply:
Actually, I use this function to populate static flows for IPv6 specific protocols.